modules/test_user_dashboard/app/models/test_user_dashboard/tud_account.rb

```diff
@@ -4,16 +4,15 @@ module TestUserDashboard
   class TudAccount < ApplicationRecord
     self.ignored_columns = %w[standard available]
 
-    ID_PROVIDERS = %w[id_me dslogon mhv].freeze
+    ID_PROVIDERS = %w[idme dslogon mhv logingov].freeze
 
     validates :first_name, :last_name, :email, :gender, presence: true
     validates :email, uniqueness: true
+    validates :id_type, presence: true
+    validates :id_type, inclusion: { in: ID_PROVIDERS }
```

Comment:
A minor thing, but I would probably make this one line - `validates :id_type, presence: true, inclusion: { in: ID_PROVIDERS }`. You also validate a bunch of attributes for presence in line 9 so it could also make sense for you to move line 11 there.